docs/howto/openstack/nova/move-server-between-regions.md

```diff
@@ -0,0 +1,311 @@
+# Moving a server from one region to another
+
+This guide will show you how to move a server to a different region in {{brand}} Cloud.
```

Comment:
With #81 having landed, please change `{{brand}} Cloud` to just `{{brand}}` when you next rebase on current `main`.